lib/kino/table.ex

```diff
@@ -49,7 +40,7 @@ defmodule Kino.Table do
               {:ok,
                %{
                  columns: list(column()),
-                 data: columnar() | row_based(),
+                 data: {:columnar | :row_based, list(list(String.t()))},
```

Comment:
Let's match the wording in `Table.Reader`

```suggestion
                 data: {:columns | :rows, list(list(String.t()))},
```